modules/weko-search-ui/weko_search_ui/utils.py

```diff
@@ -1118,19 +1127,24 @@ def check(index_ids, index_names, parent_id=0, isRoot=False):
                     _('Specified {} does not match with existing index.')
                     .format('POS_INDEX'))
         elif index_name:
+            index = None
             index = Indexes.get_index_by_name(
                 index_name, parent_id)
             if not index:
-                index = None
-                warnings.append(
-                    _('Specified {} and {} do not exist in system.').format(
-                        'IndexID', 'POS_INDEX')
-                )
+                if index_id:
+                    errors.append(_('The specified {} does not exist in system.')
```

Comment:
Hi,
Define a constant instead of duplicating this literal 'The specified {} does not exist in system.' 3 times.